PerfettoCds/Pipeline/DataOutput/PerfettoFrameEvent.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Microsoft.Performance.SDK;
+using System;
+using Utilities;
+
+namespace PerfettoCds.Pipeline.DataOutput
+{
+    /// <summary>
+    /// An event that represents a frame that was scheduled or actual displayed by a process. 
+    /// </summary>
+    public class PerfettoFrameEvent
+    {
+
+        public string FrameType { get; }
```

Comment:
Is this needed or we can sub-class this with an ExpectedFrameTimeLineSlice and ActualFrameTimeLineSlice?

Reply:
Per our discussion, leaving as one class for code simplicity.